bfg-library/src/main/scala/com/madgag/git/bfg/GitUtil.scala

```diff
@@ -48,11 +48,11 @@ trait CleaningMapper[V] extends Cleaner[V] {
 
 object GitUtil {
   
-  val ProbablyNoNonFileObjectsOverSizeThreshold = 1024 * 1024
+  val ProbablyNoNonFileObjectsOverSizeThreshold: Long = 1024 * 1024
   
-  def tweakStaticJGitConfig(massiveNonFileObjects: Option[Int]) {
+  def tweakStaticJGitConfig(massiveNonFileObjects: Option[Long]) {
     val wcConfig: WindowCacheConfig = new WindowCacheConfig()
-    wcConfig.setStreamFileThreshold(massiveNonFileObjects.getOrElse(ProbablyNoNonFileObjectsOverSizeThreshold))
+    wcConfig.setStreamFileThreshold(massiveNonFileObjects.getOrElse(ProbablyNoNonFileObjectsOverSizeThreshold).toInt)
```

Comment:
Rather than use toInt, probably best to use saturatedCast so that setStreamFileThreshold can't get passed a negative value. 


https://google.github.io/guava/releases/19.0/api/docs/com/google/common/primitives/Ints.html#saturatedCast(long)
https://stackoverflow.com/q/4355303/438886

Reply:
Good, done.